<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="RIDER-WORKBENCH" failures="1" tests="1" name="Default test" time="36.787" errors="0" timestamp="2022-10-19T19:40:14 IST">
  <testcase classname="TestCase.TitlePage" name="AddNewCustomerMale_03" time="8.756">
    <failure type="java.io.FileNotFoundException" message=".\ScreenShots\New_Customer.png (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: .\ScreenShots\New_Customer.png (The system cannot find the path specified)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:184)
at org.openqa.selenium.io.FileHandler.copyFile(FileHandler.java:136)
at org.openqa.selenium.io.FileHandler.copy(FileHandler.java:115)
at TestObject.PageObject.Screenshot(PageObject.java:241)
at TestCase.TitlePage.AddNewCustomerMale_03(TitlePage.java:136)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddNewCustomerMale_03 -->
  <testcase classname="TestCase.TitlePage" name="NewCustomer_02" time="0.0">
    <ignored/>
  </testcase> <!-- NewCustomer_02 -->
  <testcase classname="TestCase.TitlePage" name="Login_01" time="0.0">
    <ignored/>
  </testcase> <!-- Login_01 -->
  <testcase classname="TestCase.TitlePage" name="AddNewCustomerFemale_04" time="0.0">
    <ignored/>
  </testcase> <!-- AddNewCustomerFemale_04 -->
</testsuite> <!-- Default test -->
